### ISSUE TITLE
[Errno 2] No such file or directory

### ISSUE THREAD

**Issue by adion** [uipoet/sublime-jshint#14]
OS X 10.6.8—I'm still having the issue described by https://github.com/uipoet/sublime-jshint/issues/9

I followed the install instructions (installing via npm and then Package Control) and just can't seem to get this to work. FWIW, jshint works just fine on the command line.

**Comment by nikoloza** [comment]
@jamesalley `sudo npm install -g jshint` helped me, many thanks!!!

**Comment by pherris** [comment]
You should never have to `sudo npm install` stuff - fix the permissions problem and run without `sudo.`

**Comment by seanrucker** [comment]
Guess this doesn't work with nvm? My path is /Users/seanrucker/.nvm/versions/node/v0.12.7/bin/jshint

**Comment by jhenaoz** [comment]
i have the same issue with nvm, any idea how to solve it?

**Comment by jlippold** [comment]
If you're using nvm, you'll have to edit the script by hand each time you upgrade node.

First install jshint globally `npm install jshint -g`
Find the path to your node version: `which node` 
which returns `/Users/XXX/.nvm/versions/node/v5.1.0/bin/node`

Then edit: ~/Library/Application Support/Sublime Text 2/Packages/JSHint/JSHint.py https://github.com/uipoet/sublime-jshint/blob/master/JSHint.py#L25

to add that node directory
```
args['path'] = "/usr/local/share/npm/bin:/usr/local/bin:/opt/local/bin:/Users/XXX/.nvm/versions/node/v5.1.0/bin"
```

**Comment by apendua** [comment]
@jlippold I am wondering if there's any good reason not to use the value from `os.getenv('PATH');`?

**Comment by geoidesic** [comment]
I tried all of these suggestions – none of them worked for me on OS X. Installed it globally via npm and it still could be run by Sublime Text 3. Tried installing it via brew (as per the README) but brew says there's no such package available. 

If I install it globally via npm then I can run it on the command line but not via Sublime Text 3.

I thought @jlippoid was onto something there but that suggestion didn't work for me either... it just changed the error to: "env: node: No such file or directory"
